spec/integration/.kitchen.yml

```diff
@@ -0,0 +1,19 @@
+---
+driver:
+  name: docker
+  image: centos:6.7
+  use_sudo: true
```

Comment:
Ah awesome! Apparently you can insert ERB snippets into your test-kitchen config.

So re-writing the `.kitchen.yml` as follows should work for our needs:
```
--- a/spec/integration/.kitchen.yml
+++ b/spec/integration/.kitchen.yml
@@ -2,7 +2,7 @@
 driver:
   name: docker
   image: centos:6.7
-  use_sudo: true
+  use_sudo: <%= ENV['DOCKER_SUDO'] %>
```

Unless a user has `DOCKER_SUDO` environment variable set, this will default to false. Then inside our `.travis.yml` we will just add an `env` heading to set the variable.

```
--- a/.travis.yml
+++ b/.travis.yml
@@ -7,3 +7,5 @@ rvm:
 sudo: required
 services:
   - docker
+env:
+  - DOCKER_SUDO=true
```

And this should just "all work"